simperby/src/node/stf.rs

```diff
@@ -0,0 +1,288 @@
+//! The definition of the state transition function which deduces
```

Comment:
```suggestion
/// The definition of the state transition function which deduces
```
Or is this a special format for comments that come at the very top of the source code?